api/milvuspb/milvus.pb.go

```diff
@@ -105,6 +105,76 @@ func (OperatePrivilegeType) EnumDescriptor() ([]byte, []int) {
 	return fileDescriptor_02345ba45cc0e303, []int{2}
 }
 
+type CheckHealthRequest struct {
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *CheckHealthRequest) Reset()         { *m = CheckHealthRequest{} }
+func (m *CheckHealthRequest) String() string { return proto.CompactTextString(m) }
+func (*CheckHealthRequest) ProtoMessage()    {}
+func (*CheckHealthRequest) Descriptor() ([]byte, []int) {
+	return fileDescriptor_02345ba45cc0e303, []int{0}
+}
+
+func (m *CheckHealthRequest) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_CheckHealthRequest.Unmarshal(m, b)
+}
+func (m *CheckHealthRequest) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_CheckHealthRequest.Marshal(b, m, deterministic)
+}
+func (m *CheckHealthRequest) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_CheckHealthRequest.Merge(m, src)
+}
+func (m *CheckHealthRequest) XXX_Size() int {
+	return xxx_messageInfo_CheckHealthRequest.Size(m)
+}
+func (m *CheckHealthRequest) XXX_DiscardUnknown() {
+	xxx_messageInfo_CheckHealthRequest.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_CheckHealthRequest proto.InternalMessageInfo
+
+type CheckHealthResponse struct {
```

Comment:
If not healthy, should we give a reason of failed?

Reply:
We just use GetComponentStates API which is similar to Ping-Pong mode, it is a little bit difficult to give the user an available reason currently.   Maybe adding the reason is more useful after we have a more accurate usability check method?